test/decayAmplitude/testAmplitudeTree.cc

```diff
@@ -95,7 +88,7 @@ main()
 		inFile->GetObject("test", tree);
 		amplitudeTreeLeaf* ampLeaf = 0;
 		tree->SetBranchAddress("amp", &ampLeaf);
-		for (unsigned int i = 0; i < tree->GetEntriesFast(); ++i) {
+		for (long i = 0; i < tree->GetEntriesFast(); ++i) {
```

Comment:
`GetEntriesFast` returns `Long64_t`, which is `long long` not `long`. Those two are different on Windows systems (see [here](https://de.cppreference.com/w/cpp/language/types)).
I would suggest to use the ROOT type `Long64_t`.

Reply:
Done.